hw/ip/pwm/data/pwm_testplan.hjson

```diff
@@ -0,0 +1,110 @@
+                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
+  entries: [
+    {
+      name: smoke
+      desc: '''
+            Smoke test accessing a major datapath within the pwm
+
+            Stimulus:
+              - Program pwm enable register (PWM_EN)
+              - Program configuration registers (CFG, INVERT)
+              - Disable blink and heartbeat enable registers (these mode are verified with other tests)
+              - Program REGEN to lock (enable) register settings and start pwm function
+
+            Checks:
+              - Ensure the output pulses are correctly modulated for all channels
+            '''
+      milestone: V1
+      tests: ["pwm_smoke"]
+    }
+    {
+      name: perf
+      desc: '''
+            Checking ip operation at min/max bandwidth
+
+            Stimulus:
+              - Program timing registers (CLK_DIV, DC_RESN) to high/low values (slow/fast data rate)
+              - Program other required registers for pwm operation
+              - Start pwm channels
+
+            Checks:
+              - Ensure the output pulses are correctly modulated for all channels
+            '''
+      milestone: V2
+      tests: ["pwm_perf"]
+    }
+    {
+      name: blink
+      desc: '''
+            Test the blinking mode of pwm
+
+            Stimulus:
+              - Program pwm enable register (PWM_EN)
+              - Program configuration registers (CFG, INVERT)
+              - Enable blink mode (BLINK_EN) and program DUTY_CYCLE register
+              - Program REGEN to lock (enable) register settings and start pwm function
+
+            Checks:
+              - Ensure the output pulses are correctly modulated for all channels
+              - Ensure blinking channel toggles by two duty cycle values programmed to DUTY_CYCLE register
+            '''
+      milestone: V2
+      tests: ["pwm_blink"]
+    }
+    {
+      name: heartbeat
+      desc: '''
+            Test the heartbeat mode of pwm
+
+            Stimulus:
+              - Program pwm enable register (PWM_EN)
+              - Program configuration registers (CFG, INVERT)
+              - Enable heartbeat mode (HTBT_EN) and programm BLINK_PARAM register
+              - Program REGEN to lock (enable) register settings and start pwm function
+
+            Checks:
+              - Ensure the output pulses are correctly modulated for all channels
+              - Ensure output duty cycle linearly increments and decrements between two
+                blink-rate values programmed to BLINK_PARAM register
+            '''
+      milestone: V2
+      tests: ["pwm_heartbeat"]
+    }
+    {
+      name: clock_domain
+      desc: '''
+            TBD -Verify the function of pwm in both bus and core clock domains
+
+            Stimulus:
+              - Start pwm transaction in both bus and core clock domain
```

Comment:
I'm not sure what this means?  All transactions are started in the bus clock domain.   The register values are then fed through a CDC and executed in the core clock domain.

Reply:
@martin-lueker I would change that to TBD and get back later. Could you put a waveform for CDC if possible?

Reply:
Okay.